langchain4j-watsonx/src/main/java/dev/langchain4j/model/watsonx/WatsonxChat.java

```diff
@@ -145,6 +150,11 @@ abstract static class Builder<T extends Builder<T>> extends WatsonxBuilder<T> {
         private List<ChatModelListener> listeners;
         private ChatRequestParameters defaultRequestParameters;
         private Set<Capability> supportedCapabilities;
+        private Set<String> guidedChoice;
+        private String guidedRegex;
+        private String guidedGrammar;
+        private Float repetitionPenalty;
```

Comment:
```suggestion
        private Double repetitionPenalty;
```
Please use double wherever possible

Reply:
Done